packages/dd-trace/test/plugins/helpers.js

```diff
@@ -15,6 +15,7 @@ function resolveNaming (namingSchema) {
 
 function expectSomeSpan (agent, expected, timeout) {
   return agent.use(traces => {
+    const replacer = (k, v) => typeof v === 'bigint' ? Number(v) : v
```

Comment:
loss of precision, if some people debug values they're gonna be confused as to why the error is showing a weird value no ?

Reply:
If it has to be JSON formatted, there is no way around that. We could return a string instead. But that might also be confusing?